From 2560c3e2ebb5b55ab9df4b43cd8e22c9bead03bc Mon Sep 17 00:00:00 2001
From: Sebastien Bacher <seb128@ubuntu.com>
Date: Wed, 11 Jan 2023 16:56:51 +0100
Subject: [PATCH] testsuite: set the new test-language-specs as not-installable

it's intended to be used as installed tests but in CI instead
---
 testsuite/meson.build | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

Index: gtksourceview4/testsuite/meson.build
===================================================================
--- gtksourceview4.orig/testsuite/meson.build	2023-01-12 08:59:47.741505549 +0100
+++ gtksourceview4/testsuite/meson.build	2023-01-12 09:10:00.669672844 +0100
@@ -26,7 +26,7 @@
   ['test-iter'],
   ['test-language'],
   ['test-languagemanager'],
-  ['test-language-specs'],
+  ['test-language-specs', false],
   ['test-mark'],
   ['test-printcompositor'],
   ['test-regex'],
@@ -42,6 +42,7 @@
 
 foreach test: testsuite_sources
   test_name = test.get(0)
+  install_test = test.get(1, true) and get_option('install_tests')
   test_sources = gtksource_res + [
     '@0@.c'.format(test_name),
   ]
@@ -60,13 +61,13 @@
   test_exe = executable(test_name, test_sources,
           c_args: testsuite_c_args,
     dependencies: [core_dep, completionwords_dep],
-         install: get_option('install_tests'),
+         install: install_test,
      install_dir: testexecdir
   )
 
   test(test_name, test_exe, suite: package_string, env: test_env)
 
-  if get_option('install_tests')
+  if install_test
     test_t = configuration_data()
     test_t.set('testexecdir', testexecdir)
     test_t.set('test', test_name)
